packit_service/worker/events/pagure.py

```diff
@@ -58,7 +63,7 @@ def __init__(
         comment: str,
         commit_sha: str = "",
     ):
```

Comment:
Why not adding  `comment_object: Optional[Comment] = None,` also here?

Reply:
the thing is that Pagure API doesn't support getting comment by its id so we can't get comment_object for pagure